src/components/auth/login/GoogleAndFacebookLogIn.jsx

```diff
@@ -14,13 +14,9 @@ export default function GoogleAndFacebookLogIn() {
       imageUrl: user.photoURL,
       isAdmin: false,
     };
-    const createNewSubCollection = await db
-      .collection("users")
-      .doc(info.userId)
-      .collection("liked")
-      .add({
-        liked: 0,
-      });
+    await db.collection("users").doc(info.userId).collection("liked").add({
+      liked: 0,
```

Comment:
there should not be any changes for signup and facebook login in a PR titled "blog overview."

please split them out into a separate branch

Reply:
i just deleted a variable that was not used and because of it i couldn't pass netlify's test thats way i deleted it , should i create a new branch for this  and then pr again ?

Reply:
yes